jenkins/scripts/slurm_install.sh

```
@@ -20,18 +17,28 @@ slurm_install_setup() {
+                retry_command --timeout 2700 pip3 install --retries 10 "ray[default]==2.55.1"
+            fi
+            retry_command --timeout 2700 bash -c "pip3 install --retries 10 opencv-python-headless"
+            llmSrcNode=$resourcePathNode/TensorRT-LLM/src
+            retry_command --timeout 2700 bash -c "cd $llmSrcNode && pip3 install --retries 10 -r requirements-dev.txt"
+            retry_command --timeout 2700 bash -c "cd $resourcePathNode && pip3 install --retries 10 --force-reinstall --no-deps TensorRT-LLM/tensorrt_llm-*.whl"
         fi
-        retry_command --timeout 2700 bash -c "pip3 install --retries 10 opencv-python-headless"
-        retry_command --timeout 2700 bash -c "cd $llmSrcNode && pip3 install --retries 10 -r requirements-dev.txt"
-        retry_command --timeout 2700 bash -c "cd $resourcePathNode && pip3 install --retries 10 --force-reinstall --no-deps TensorRT-LLM/tensorrt_llm-*.whl"
+        llmSrcNode=$resourcePathNode/TensorRT-LLM/src
+        cd $resourcePathNode
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)